a7/validate.py

```diff
@@ -25,7 +26,15 @@ def validate_py3(path_to_module):
         # cf https://www.python.org/dev/peps/pep-0328/
         linter.disable("no-absolute-import")
         with fix_import_path([path_to_module]):
-            linter.check(path_to_module)
+            syspath = sys.path[:]
+            try:
+                # Remove site-packages from imports. It keeps the standard
+                # library, but it prevents pylint from parsing potentially big
+                # third party modules.
+                sys.path = [p for p in syspath if 'site-packages' not in p]
```

Comment:
I feel like this behavior should be behind a command flag.

Reply:
I hear you but If we want to get that in early, that means making an agent change to get the better performance.

And it **shouldn't** change behavior. If it does, we need to fix it, not have a flag to revert back to the old one.